src/runner.rs

```diff
@@ -1,3 +1,8 @@
+//! The runner module handles retrieving Runners. A Runner is a user with at least one Run tied to
+//! their Splits.io account.
```

Comment:
Is this true? So there can be a user that is not a runner?!

Reply:
No idea, but that's what the docs say. Would probably be a good idea to ask in the Splits.io Discord server.

Reply:
Yes this is true, a [Runner][1] is defined as a user with >= 1 run, so users without runs aren't discoverable using the API.

This is weird behavior that we wouldn't opt for again today, but we also won't break. It was originally because the API couldn't do anything with a user except look up runs, so we made it appear to imitate the (at the time) game and category behavior of self-destructing when becoming childless.

It's possible that in the future we may add a parallel User model that doesn't have this behavior and silently deprecate Runner without breaking it. But right now that's not high on our list of priorities.

[1]: https://github.com/glacials/splits-io/blob/master/docs/api.md#runner